README.md

```diff
@@ -64,10 +68,33 @@ DXVK 1.10 and older does not support `DXVK_ENABLE_NVAPI`. Disable the `nvapiHack
+Using DXVK-NVAPI with other GPU vendors / drivers has very limited use. Outside of testing, only Reflex (LatencyFlex) or HDR entry points provide benefits. This requires DXVK to see the GPU as an NVIDIA GPU. Use `DXVK_CONFIG="dxgi.hideAmdGpu = True"` to spoof an AMD GPU as NVIDIA GPU. Use `DXVK_CONFIG="dxgi.customVendorId = 10de"` for generally spoofing an NVIDIA GPU.
 
 Setting `DXVK_NVAPI_ALLOW_OTHER_DRIVERS=1` is needed for successful DXVK-NVAPI initialization when using a driver other than the NVIDIA proprietary driver or Mesa NVK. The reported driver version on drivers other than the NVIDIA proprietary driver will be 999.99. Overriding the reported driver version is still recommended. The reported GPU arrchitecture for other vendors is always Pascal to prevent attempts to initialize DLSS. This behavior cannot be changed without modifying the source code.
 
+### Vulkan Reflex layer
+
+The Vulkan Reflex layer, when built as a Linux-side layer, is not installed into Wine prefix. Instead it's installed like any other Vulkan implicit layer. It is distributed as a pair of files:
+
+- `VkLayer_DXVK_NVAPI_reflex.json` (the layer manifest)
+- `libdxvk_nvapi_vkreflex_layer.so` (the layer library)
+
+To make it discoverable by your Vulkan loader:
+
+1. Do one of the following:
+    - Install the layer manifest into one of default implicit layer search paths (e.g. `${XDG_DATA_HOME}/vulkan/implicit_layer.d` where Steam usually installs its layers)
+    - Export full path to the directory containing the layer manifest in `VK_ADD_IMPLICIT_LAYER_PATH` environment variable (e.g. `VK_ADD_IMPLICIT_LAYER_PATH="${HOME}/dxvk-nvapi/layer"`)
+        - Note that `VK_ADD_IMPLICIT_LAYER_PATH` works only with Vulkan loader v1.3.296 or newer
+2. If needed, edit the layer manifest to ensure that its `library_path` points to the layer library
+     - For example, if the layer library is in the same directory as the layer manifest, the path should be `./libdxvk_nvapi_vkreflex_layer.so`
+     - For other scenarios where the layer library is nearby, relative paths work as well (e.g. `../../../lib/libdxvk_nvapi_vkreflex_layer.so`)
+     - It's also possible to provide full path to the layer library (e.g. `/home/user/dxvk-nvapi/layer/libdxvk_nvapi_vkreflex_layer.so`)
+     - Finally, if the layer is in one of the paths searched by dynamic linker (like `/usr/lib`, this depends on the distro), it's possible to set the library path to just the name of the layer library: `"library_path": "libdxvk_nvapi_vkreflex_layer.so"`
+
+Run `vulkaninfo` and check the list of discovered layers to verify there is an entry named `VK_LAYER_DXVK_NVAPI_reflex (DXVK-NVAPI Vulkan Reflex compatibility layer)`.
```

Comment:
I found `vulkaninfo` a but tricky for validating the layer setup, since `vulkaninfo` shows that the manifest has been found, but not that the `library_path` inside the manifest is correct. May be add the following sentence 

```
Note that `vulkaninfo` does not validate the correctness of the `library_path` property inside the manifest.
```

or something like this.

Reply:
Fair enough, but it does cause the loader to emit an error if layer was requested but the path was invalid. I'll describe this case in greater detail.